operations/observability/mixins/workspace/rules/satellite/servicemonitor.yaml

```
@@ -0,0 +1,17 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: containerd-metrics
+  labels:
+    app.kubernetes.io/component: containerd-metrics
+    app.kubernetes.io/name: containerd-metrics
+    app.kubernetes.io/part-of: kube-prometheus
+spec:
+  namespaceSelector:
+    matchNames:
+    - monitoring-satellite
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: containerd-exporter
+  endpoints:
+  - port: v1/metrics
```

```suggestion
  endpoints:
  - port: metrics
    path: v1/metrics
```

Looking at [prometheus operator docs](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#endpoint), this looks like the correct way of specifying the endpoint path.

The port name need to match the name given in the service(See also: https://github.com/gitpod-io/ops/pull/6775/files#r1024346770)